npm/packs/core/src/abp.js

```diff
@@ -773,4 +773,14 @@ var abp = abp || {};
         return abp.features.values[name];
     };
     
+    /* GLOBAL FEATURES *************************************************/
+
+    abp.globalFeatures = abp.globalFeatures || {};
+
+    abp.globalFeatures.enabledFeatures = abp.globalFeatures.enabledFeatures || {};
```

Comment:
```suggestion
    abp.globalFeatures.enabledFeatures = abp.globalFeatures.enabledFeatures || [];
```

Shouldn't be like that?

Reply:
You're right.